compiler/rustc_middle/src/query/mod.rs

```diff
@@ -827,6 +827,11 @@ rustc_queries! {
         }
     }
 
+    query mir_inliner_callers_count_map(_: ()) -> Lrc<FxHashMap<DefId, usize>> {
```

Comment:
This query only looks at the current crate, so is can only store `LocalDefId`s.

Reply:
Callees can be from other crates too.